src/NServiceBus.NHibernate/TimeoutPersisters/TimeoutEntity.cs

```diff
@@ -29,6 +29,11 @@ class TimeoutEntity
         public virtual byte[] State { get; set; }
 
         /// <summary>
+        /// Timeout endpoint name.
+        /// </summary>
+        public virtual string Endpoint { get; set; }
```

Comment:
@MarcinHoppe Add code comment that property order is used for index column order?